packages/beacon-node/src/execution/engine/types.ts

```diff
@@ -419,7 +419,7 @@ export function serializeExecutionLayerWithdrawalRequest(
 ): ExecutionLayerWithdrawalRequestRpc {
   return {
     sourceAddress: bytesToData(withdrawalRequest.sourceAddress),
-    validatorPublicKey: bytesToData(withdrawalRequest.validatorPublicKey),
+    validatorPublicKey: bytesToData(withdrawalRequest.validatorPubkey),
```

Comment:
There is a inconsistency in the execution apis, right now all EL clients besides Besu use `validatorPublicKey`, this means pairing with Besu will not work right now if execution layer withdrawals requests are submitted to the network.

For now, we should just follow the spec

I opened a PR to potentially address this inconsistency
- https://github.com/ethereum/execution-apis/pull/549